### ISSUE TITLE
okteto up --remote fails to put the terminal in raw mode

### ISSUE THREAD

**Issue by rberrelleza** [okteto/okteto#876]
**Describe the bug**
`okteto up --remote` fails. logs point at this error as the root cause:
```
command failed: request for raw terminal failed: The handle is invalid.
```

**To Reproduce**
```
Microsoft Windows [Version 10.0.18363.778]
(c) 2019 Microsoft Corporation. All rights reserved.
C:\Users\UserName\source\repos\ci-go-builder>C:\Users\UserName\AppData\Local\Programs\okteto.exe up -f c:\Users\UserName\source\repos\ci-go-builder\okteto.yml --remote 22100
 +  Development environment activated
⠙ Starting the file synchronization service...
The value of /proc/sys/fs/inotify/max_user_watches in your cluster nodes is too low.
This can affect Okteto's file synchronization performance.
                                          own-issues/index.html for more information.
 +  Files synchronized
    Namespace: default
    Name:      ci-go-builder
    SSH:       22100 -> 2222
    Forward:   2345 -> 2345
               8080 -> 8080
 x  Command execution failed
C:\Users\UserName\source\repos\ci-go-builder>
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 1.8.8

**Comment by rberrelleza** [comment]
Derek assign: rberrelleza

**Comment by kempy007** [comment]
:)

**Comment by rberrelleza** [comment]
fixed by #877
